web-config.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 web:
+  domain: null
+  basePath: null
```

Comment:
In a Java web app, `basePath` should never be set.  Should this (and `domain`) be a required configuration field?